core/lib/dal/src/blocks_dal.rs

```diff
@@ -2196,7 +2196,7 @@ impl BlocksDal<'_, '_> {
                 )
             WHERE
                 l1_batch_number IS NULL
-                AND fee_account_address = '\x0000000000000000000000000000000000000000'::bytea
+                AND fee_account_address = 'x0000000000000000000000000000000000000000'::bytea
```

Comment:
IIRC I've seen this before and it caused some troubles. I think it's a brainfart of query linter and probably should be rolled back.